docs/config_entries_config_flow_handler.md

```diff
@@ -39,8 +42,6 @@ class ExampleConfigFlow(data_entry_flow.FlowHandler):
         # Handle discovery info
 ```
 
-If the result of the step is to show a form, the user will be able to continue the flow from the config panel.
-
 ## Translations
 
```

Comment:
Do we need to explain Loaklise, or at least link to the dedicated page about localization? Also, do we need to tell people that even when keys are added to Lokalise they won't appear for a while until someone does a export?

Reply:
Done